packages/core/src/devtools/injectDevTools.ts

```diff
@@ -0,0 +1,84 @@
+import { DEFINITION_LOOKUP_TABLE, SEQUENCE_PREFIX } from '../constants';
+import { LookupItem, SequenceHash } from '../types';
+import { MK_DEBUG } from './store';
+import { DebugSequence } from './types';
+import { getDebugClassNames } from './utils';
+
+const getDebugTree = (debugSequenceHash: SequenceHash, parentNode?: DebugSequence) => {
+  const lookupItem: LookupItem | undefined = DEFINITION_LOOKUP_TABLE[debugSequenceHash];
```

Comment:
Let';s move this function to a separate file. Then you can separate tests in `packages/core/src/devtools/injectDevTools.test.ts`

- `injectDevTools` should a property to a passed `window`
- `getDebugTree` does traversal